src/embed_tests/TestPyFloat.cs

```diff
@@ -25,6 +25,7 @@ public void Dispose()
         public void IntPtrCtor()
         {
             var i = new PyFloat(1);
+            Runtime.Runtime.XIncref(i.Handle);
```

Comment:
Same.
Since that might affect existing code, an overload should be added, that specifies ownership